components/blitz/src/omero/cmd/graphs/DuplicateI.java

```diff
@@ -414,7 +414,7 @@ public Object apply(Object original) {
                         }
                     }
                     /* process property values that do not relate to edges in the model object graph */
-                    for (final String property : graphPathBean.getSimpleProperties(superclassName)) {
+                    for (final String property : graphPathBean.getSimpleProperties(superclassName, false)) {
```

Comment:
Is it worth defining a `GraphPathBean.getSimpleProperties(String)` signature which would default to `GraphPathBean.getSimpleProperties(String, false)`?